platform/admin/core/admin-core-logviewer/pom.xml

```diff
@@ -79,7 +79,7 @@
                 <configuration>
                     <instructions>
                         <Web-ContextPath>/admin/logviewer</Web-ContextPath>
-                        <_wab>${project.build.directory}/webapp/</_wab>
```

Comment:
See my comment above. That applies to this as well. Without this line, the plugin won't build a WAB.